src/label_studio_sdk/converter/brush.py

```diff
@@ -116,7 +116,7 @@ def decode_from_annotation(from_name, results):
         width = result["original_width"]
         height = result["original_height"]
         labels = result[key] if key in result else ["no_label"]
-        name = from_name + "-" + "-".join(labels)
```

Comment:
if you have multiple brushlabel control tags, your brushlabels will contain only one of them.

Reply:
I included an example in the next comment where I used the attached labeling interface (XML below).  I was able to define two brushlabel classes and exported masks contained both classes.   The first 3 exported image masks were of type "Hotdog" and the last 3 exported image masks were of type "Not Hotdog".  I've used keypoints, rectangles, and brush labels to annotate and all were exported as masks with the proper class name.

I'm not sure if this is what you mean though by multiple brushlabel control tags.

<View>
  <Image name="image" value="$image" zoom="true"/>
  <Header value="Keypoint Labels"/>
  <KeyPointLabels name="tag2" toName="image" smart="true">
    <Label value="Not Hotdog" smart="true" background="#00FF00" showInline="true"/>
    <Label value="Hotdog" smart="true" background="#FF0000" showInline="true"/>
  </KeyPointLabels>
  <Header value="Rectangle Labels"/>
  <RectangleLabels name="tag3" toName="image" smart="true">
    <Label value="Not Hotdog" smart="true" background="#00FF00" showInline="true"/>
    <Label value="Hotdog" smart="true" background="#FF0000" showInline="true"/>
  </RectangleLabels>
  <Header value="Brush Labels"/>
  <BrushLabels name="tag" toName="image">
    <Label value="Not Hotdog" smart="true" background="#00FF00" showInline="true"/>
    <Label value="Hotdog" smart="true" background="#FF0000" showInline="true"/>
  </BrushLabels>
</View>